geoips/plugins/modules/readers/ahi_hsd.py

```diff
@@ -1103,6 +1198,7 @@ def call(
     xarray_obj.attrs["start_datetime"] = start_dt
     xarray_obj.attrs["end_datetime"] = end_dt
     xarray_obj.attrs["source_name"] = "ahi"
+    xarray_obj.attrs["source_file"] = fnames[0]
```

Comment:
Please see xarray standards: https://github.com/NRLMMD-GEOIPS/geoips/blob/main/docs/source/devguide/xarray_standards.rst#xarray-standard-attributes

Reply:
Gotcha. I changed this to adhere to the xarray_standards you sent.

Reply:
Maybe these are relevant, are there others?
* https://github.com/NRLMMD-GEOIPS/geoips/pull/427/commits/d4eeb2855ef29953f07f51718f39d6326079c443
* https://github.com/NRLMMD-GEOIPS/geoips/pull/427/commits/7ce0ecb8ab2382898f47a885fcb3b6dc3389ba39

Reply:
[Here](https://github.com/NRLMMD-GEOIPS/geoips/pull/427/files#r1821413678) is where it was resolved.

Reply:
Thanks!